modules/sandboxed-containers-enable-debug-crio.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/troubleshooting-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-enable-debug-logs_{context}"]
+= Enabling debug logs for {sandboxed-containers-first}
+
+As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
+
+.Procedure
+
+To change the `log_level` in CRI-O:
+
+. Create a YAML file for the `ContainerRuntimeConfig` CR with the following manifest:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: crio-debug
+spec:
+ machineConfigPoolSelector:
+  matchLabels:
+    pools.operator.machineconfiguration.openshift.io/worker: '' <1>
```

Comment:
I don't know how it works within the actual docs, but adding the '<1>' wouldn't mess the yaml file when one is copying it? can we add '#' before so that it would be a comment yaml wise?

Reply:
The <1> doesn't appear in the file when it renders in the help. It shows up as a small icon.

I just copied it from the preview - here's the paste:

apiVersion: machineconfiguration.openshift.io/v1
kind: ContainerRuntimeConfig
metadata:
 name: crio-debug
spec:
 machineConfigPoolSelector:
  matchLabels:
    pools.operator.machineconfiguration.openshift.io/worker: '' 
 containerRuntimeConfig:
    logLevel: debug